fast64_internal/oot/oot_utility.py

```diff
@@ -874,6 +874,27 @@ def _eval(node):
     return f"0x{_eval(node.body):X}"
 
 
+def getShiftFromMask(mask: int):
+    """Returns the shift value from the mask"""
+
+    # get the shift by subtracting the length of the mask
+    # converted in binary on 16 bits (since the mask can be on 16 bits) with
+    # that length but with the rightmost zeros stripped
+    return int(f"{len(f'{mask:016b}') - len(f'{mask:016b}'.rstrip('0'))}", base=10)
```

Comment:
what is the difference with
```suggestion
    return len(f'{mask:016b}') - len(f'{mask:016b}'.rstrip('0'))
```
or even
```suggestion
    return len(f'{mask:b}') - len(f'{mask:b}'.rstrip('0'))
```

I would also add an assert the mask *is* a mask, i.e. that `f'{mask:b}'.rstrip('0')` looks like "1111" and not "11101" or "":
```
mask_binary_str = f"{mask:b}"
assert set(mask_binary_str.rstrip("0")) == {"1"}, mask_binary_str
```

Reply:
I wrote that function 2 years ago I can't remember how it works

Reply:
I'm not a fan of the second option so I'll keep the ``016b``, I will also add the assert just in case